src/fsharp/FSharp.Core/set.fsi

```diff
@@ -33,7 +39,13 @@ type Set<[<EqualityConditionalOn>]'T  when 'T : comparison> =
     ///
     /// <returns>The result set.</returns>
     /// 
-    /// <example-tbd></example-tbd>
+    /// <example>
+    /// <code lang="fsharp">
+    /// let set = Set.empty.Add(1).Add(1).Add(2)
+    /// printfn "The new set is: %A" set
```

Comment:
Shall we use interpolated strings in examples?

Reply:
I used the simpler one so that the output will display all the values compared to interpolated strings where it cuts it out and outputs `[1; 2; 3; ...]` vs `[1;2;3;4;]`

Reply:
Please use interpolated strings